mne/conftest.py

```diff
@@ -163,6 +163,12 @@ def pytest_configure(config):
         warning_line = warning_line.strip()
         if warning_line and not warning_line.startswith("#"):
             config.addinivalue_line("filterwarnings", warning_line)
+    # TODO: Fix this with casts?
+    # https://github.com/numpy/numpy/pull/22449
+    try:
+        np.set_printoptions(legacy="1.25")
```

Comment:
CI failure suggests that this should actually be an integer:

```suggestion
        np.set_printoptions(legacy=125)
```

even though the example shows it as a string `"1.25"`

Reply:
Apparently that's a newer option not compat with old versions, so I just `check_version` it instead now. That way we can use the string, which seems nicer than the int anyway. (No idea if the old versions will understand a request to use a `legacy` version that is *newer* than their version... ideally they'd ignore it but I didn't feel like finding out the hard way.)